OdpDescriptor.cs

```diff
@@ -144,12 +144,13 @@ public override void SetParametersOnCommand(IDbCommand cmd, IEnumerable paramete
 
                 string paramName = DbCaches.ParamNames.Get(x);
                 
-                OracleParameter oracleParam;
-                if (addMe != DBNull.Value && addMe.GetType().GetProperty("GeometryType") != null)
-                {
-                    oracleParam = new OracleParameter(paramName, OracleDbType.Object, addMe.ToString().Length, addMe.ToString());
-                    oracleParam.UdtTypeName = "SDE.ST_GEOMETRY";
-                }
+                OracleParameter oracleParam;
+                if (addMe != DBNull.Value && addMe is string && ((string)addMe).Length > 3999)
```

Comment:
I'll run some queries on the parcel data to see where about how long these usually are.